src/Build.UnitTests/BackEnd/BuildResult_Tests.cs

```diff
@@ -10,19 +10,22 @@
 using Microsoft.Build.Framework;
 using Microsoft.Build.Shared;
 using Microsoft.Build.Unittest;
+using Shouldly;
 using Xunit;
-
+using Xunit.Sdk;
```

Comment:
```suggestion
using Xunit.Abstractions;
```

Reply:
TestOutputHelper is defined in Xunit.Sdk

Reply:
Got it, switched to using ITestOutputHelper